Report.md

```diff
@@ -1,17 +1,104 @@
-# Sean's Report
-
-This is an edit to test
-This is also an edit test :D
-
-1. Outline the objectives of the project and define the problem (Steps 1 & 2 of Design Thinking
-Methodology).
-2. Identify stakeholders.
-3. Describe your approach from a Design Thinking principles perspective.
-4. Explain the agile methodology, namely, Scrum you have used to carry out the procedure. In
-your explanation, ensure that you outline activities from Scrum that you use.
-5. List assumptions you have made in the systems analysis.
-6. Document your proposed workproducts and models.
-7. Discuss the competitive advantages might be gained in developing the new information
-system. Identify and discuss the possible adverse effects for this Business if its information
-system project fails
-8. Document in GitHub all your iteration and models as you progress.
+# Information System Development Methodologies Project Report
+
+## 1. Outline the objectives of the project and define the problem 
+
+
+
+## 2. Identify Stakeholders 
+
+#### Travel Company
+ * *Who are we empathising with?*
+   * A travel company who provides holiday packages  
+ * *What do they need to do?*
+   * Would like to develop an information system to improve in-house call management centre  (CMC).
+   * This CMC system will:
+     * Provide improved call routing and dynamic call flow control for both inbound and outbound calls
+     * Adjust the call flow rate so that suitable relationship managers (RMs) are matched according to RMs skills and customer profiles 
+     * Match customers with informed RMs via profiles 
+     * Segment customers into social and cultural segements according to postcodes and surnames - via the "Profiler Tool"
+     * CMC system will adjust RMs preformance in selling packages and serving customers effectively and efficiently to rank RMs
+ * *What do they see?*
+    * They see the packages they have to sell - and the destination
+    * They see the customers their targeting
+    * The performance of the RMs
+    * Sales figures on certain packages 
+ * *What do they say?*
+    * Customers with the highest scores are served first
+    * Calls are directed to RMs based on skill levels and best match
+ * *What do they do?*
+    * The travel company employs RMs to dial numbers according to a customer target list from a database
+    * Repeat customers may have a higher score, customers from particular postcodes may have lower scores
+ * *What do they hear?*
+    * They hear the feedback of customers 
+ * *What do they think and feel?*
+    * Fears on losing customer’s based on RMs performance
+    * Gain the sale of travel packages 
```

Comment:
They want customer details and preferences  (customer profile) to always be up to date  so that  can correctly match them to a holiday package and ensure they have a positive customer experience